doc/releasenotes.rst

```diff
@@ -3,6 +3,136 @@
 *************
 Release Notes
 *************
+0.7.x
+=====
+* Add Python 3 compatibility.
+
+See also :download:`CHANGELOG` for more details on changes.
+
+Important changes for Python 3 compatibility
+-------------------------------------------
+
+General notes
+^^^^^^^^^^^^^
+
+The package is now compatible with both Python 2 and 3.  The goals of the
+migration were:
+* Do not change the public API.
+* Do not break existing functionality for Python 2.
+* Ease migration of packages using katcp to Python 3.
+
+Despite these goals, some of the stricter type checking that has been added
+may force minor updates in existing code.  E.g., using integer for the options
+of a discrete sensor is no longer allowed.
+
+Asynchronous code is still using `tornado in the same Python 2 way.  The new
+Python 3.5 ``async`` and ``await`` keywords are not used.  The tornado version
+is also pinned to an old version that supports both Python 2 and 3.
```

Comment:
Tornado 5 still supports Python 2 - is there some other reason you're pinning it to `<5`?